qds_sdk/clusterv2.py

```diff
@@ -509,6 +517,12 @@ def cluster_info_parser(argparser, action):
                                    default=None,
                                    help="overrides for airflow cluster", )
 
+        internal_group = argparser.add_argument_group("internal settings")
```

Comment:
Should the internal settings be exposed to user? I believe for internal settings, one requires super admin access which an outside user won't have.